tools/coverity.sh

```diff
@@ -1,228 +1,62 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
Looking at ff12a7954c1 I think it would be better to keep `/usr/bin/env` since the other scripts use it.